src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -90,7 +93,9 @@ def sendAlert(self, alertName, severity, summary, description, service, tag="wmc
         # need to do this because pycurl_manager only accepts dict and encoded strings type
         params = json.dumps(request)
 
+        # provide dump of alert send to AM which will allow to match it in WM logs
         res = self.mgr.getdata(self.alertManagerUrl, params=params, headers=self.headers, verb='POST')
+        self.logger.info("ALERT UUID %s, status=%s", params, labels["uuid"])
```

Comment:
This line looks strange to me. I think the order of the arguments is wrong; but in addition, did you want to print the whole content of the alert? If so, then I think you should rephrase what will be recorded in the log instead of "status".

Reply:
thanks for spotting this, left over after initial code. I took this opportunity to obtain header from the request and modified logger printout to print alert name, its uuid and http code status.